### ISSUE TITLE
Stderr: VBoxManage: error: Could not rename the directory on OSX

### ISSUE THREAD

**Issue by jcanfield** [ChrisWiegman/Primary-Vagrant#49]
So this is probably something to do with Vagrant. I followed the 3 simple steps to install. Did a `git clone` then `vagrant up`. I was halted when running default: Running 'pre-boot' VM customizations.

Below is the output of my Terminal. Also, I browsed the other Issues, I did not see this as a duplicate.

Thanks -- Josh

```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'ubuntu/trusty64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'ubuntu/trusty64' is up to date...
==> default: A newer version of the box 'ubuntu/trusty64' is available! You currently
==> default: have version '20150814.0.1'. The latest is version '20150909.1.0'. Run
==> default: `vagrant box update` to update.
==> default: Setting the name of the VM: PV_default_1442254687333_17260
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 => 2222 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
A customization command failed:

["modifyvm", :id, "--name", "Primary Vagrant"]

The following error was experienced:

#<Vagrant::Errors::VBoxManageError: There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["modifyvm", "9fa33314-efa8-44dd-86f6-16bb096a9349", "--name", "Primary Vagrant"]

Stderr: VBoxManage: error: Could not rename the directory '/Users/jcanfield/VirtualBox VMs/PV_default_1442254687333_17260' to '/Users/jcanfield/VirtualBox VMs/Primary Vagrant' to save the settings file (VERR_ALREADY_EXISTS)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component SessionMachine, interface IMachine, callee nsISupports
VBoxManage: error: Context: "SaveSettings()" at line 2788 of file VBoxManageModifyVM.cpp
>

Please fix this customization and try again.
```

**Comment by jcanfield** [comment]
Okay, so not really sure why I recieved that error. I only ran `vagrant up` once yet there were two instances of PV in my $HOME/VirtualBox VMs folder. Odd.

Just ran a simple:
```
rm -Rfv "~/VirtualBox VMs/Primary Vagrant"
rm -Rfv "~//VirtualBox VMs/V_default_1442254687333_17260"
rm -Rfv "./PV/.vagrant"
```

NOTE: Just removed the PV folders from my Virtual Box VMs folder then went back into the project and went to PV then removed the `.vagrant` folder.

After that everything ran just fine.
